cognite_toolkit/_cdf_tk/commands/modules.py

```diff
@@ -256,6 +256,35 @@ def init(
 
         raise typer.Exit()
 
+    def _select_modules_in_package(self, package: Package) -> list[ModuleLocation]:
+        choices = []
+        dependencies: list[str] = []
```

Comment:
There could be repeating dependencies so lets use a set
```suggestion
        dependencies: set[str] = set()
```